tests/basic/tests.py

```diff
@@ -202,6 +202,17 @@ def test_microsecond_precision_not_supported(self):
         self.assertEqual(Article.objects.get(id__exact=a9.id).pub_date,
             datetime(2005, 7, 31, 12, 30, 45))
 
+    @skipIfDBFeature('supports_microsecond_precision')
+    def test_microsecond_precision_not_supported_edge_case(self):
+        # In MySQL, microsecond-level precision isn't available. You'll lose
+        # microsecond-level precision once the data is saved.
+        a = Article.objects.create(
+            headline='Article 9',
+            pub_date=datetime(2008, 12, 31, 23, 59, 59, 999999))
```

Comment:
The usual style is to put the closing parenthesis on the next line and include a trailing comma on this line.